addons/website/static/src/systray_items/translate_website.js

```diff
@@ -14,10 +14,12 @@ class TranslateWebsiteSystray extends Component {
     startTranslate() {
         const { pathname, search, hash } = this.websiteService.contentWindow.location;
         if (!search.includes('edit_translations')) {
-            const searchParams = new URLSearchParams(search);
+            let searchParams = new URLSearchParams(search);
             searchParams.set('edit_translations', '1');
+            // Decode URI parameters: revert + to space then decodeURIComponent.
+            searchParams = decodeURIComponent(searchParams.toString().replace(/\+/g, '%20'));
             this.websiteService.goToWebsite({
-                path: encodeURI(pathname + `?${searchParams.toString() + hash}`),
+                path: encodeURI(pathname + '?' + searchParams + hash),
```

Comment:
Imo this is wrong, we should simply remove the `encodeURI`.
`goToWebsite` is expecting a non encoded path, it is itself in charge of encoding it

Reply:
Updated :+1:  Did not notice that `website.website_preview` action (called in `goToWebsite`) is in charge of re-encoding it. Thanks for the check :pray: